pkg/apis/edgesite/v1alpha1/default.go

```diff
@@ -157,6 +155,7 @@ func NewMinEdgeSiteConfig() *EdgeSiteConfig {
 				ClusterDNS:            "",
 				ClusterDomain:         "",
 				RemoteRuntimeEndpoint: constants.DefaultRemoteRuntimeEndpoint,
+				//TODO (@kuramal) auto set the value of PodSandboxImage by the architecture(x86,amd64,arm or arm64)
```

Comment:
```suggestion
				//TODO (@kuramal) Automatically set PodSandboxImage according to the architecture.(x86,amd64,arm or arm64)
```
And please move this line above L160